<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmbeddedDialog</class>
 <widget class="QDialog" name="EmbeddedDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>239</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Embedded Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Food Name:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Price:</string>
     </property>
     <property name="buddy">
      <cstring>typeEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Ingredients:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Food Type:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="foodNameEdit">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="typeEdit">
     <item>
      <property name="text">
       <string>Appetizer</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Drink</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Entree</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dessert</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QTextEdit" name="ingredientsEdit"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Image Name:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="imgNameEdit"/>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="submitButton">
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="priceEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
